medium/profiles/tests/test_services.py

```diff
@@ -0,0 +1,63 @@
+from urllib import request
+
+from django.core.files import File
+from django.test import TestCase
+
+from medium.profiles.services import ProfileService
+from medium.users.models import User
+
+
+class ProfileServiceTestCase(TestCase):
+    def setUp(self):
+        self.user = User.objects.create(
+            username="test_user",
+            email="test_user@dummy.com",
+            password="Test_pass123!",
+        )
+        self.test_img_url = "https://bit.ly/3vQgl0t"
+        self.img = request.urlretrieve(self.test_img_url)[0]
+        self.service = ProfileService(
+            self.user.id, File(open(self.img, "rb")), File(open(self.img, "rb"))
+        )
+
+    def tearDown(self):
+        self.user.delete()
+
+    def test_create_profile(self):
```

Comment:
PRO TIP: if there is any major validation, make sure to add test for it.

Reply:
Sounds good!

I have extended the test suite to account for more validations